content/en/docs/concepts/extend-kubernetes/compute-storage-net/resource-drivers.md

```diff
@@ -0,0 +1,280 @@
+---
+title: DRA Resource Drivers
+description: Resource drivers provide non-trivial allocation logic and management for devices or resources that require vendor-specific or just complex setup, such as GPUs, NICs, FPGAs, etc.
+content_type: concept
+weight: 10
+---
+
+<!-- overview -->
+{{< feature-state for_k8s_version="v1.26" state="alpha" >}}
+
+Kubernetes provides a
+[Dynamic Resource Allocation (DRA)](https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/3063-dynamic-resource-allocation)
```

Comment:
This should link to the Kubernetes docs on DRA (which might then mention the KEP as further reading).

Reply:
@pohly was there such a doc, or is this PR supposed to be it?

Reply:
Done.